client/landing/stepper/declarative-flow/internals/steps-repository/importer/index.tsx

```diff
@@ -133,7 +125,7 @@ export function withImporterWrapper( Importer: ImporterCompType ) {
 		function renderStepContent() {
 			if ( isLoading() ) {
 				return <LoadingEllipsis />;
-			} else if ( ! siteSlug ) {
+			} else if ( ! siteSlug || ! siteItem || ! siteId || ! fromSite ) {
```

Comment:
It's not totally clear to me if `Importer` should be rendered when missing `siteItem`, `siteId`, or `fromSite`, but looking at some instances (like [this one](https://github.com/Automattic/wp-calypso/blob/53cbe1946d639c58bfe2d7e347812f5c6d5cdcb9/client/blocks/importer/blogger/index.tsx#L53)) which use these props, it appears that they are required. The code below would therefore cause a fatal error if run when these are missing, and so here I've made them required.

Reply:
Maybe we could ask for a confirmation from a team who's more familiar with that? @Automattic/caribou perhaps?

Reply:
Yeah, agreed. I originally had figured I'd add fallbacks to the type casting below (eg: `siteId={ siteId ?? 0 }`) but that felt really wrong (although better than using `siteId as number`) which led to me this implementation instead. However, this code is very dynamic and hard for me to follow so I'm not confident of the correct approach.

Reply:
Hey guys,

I just checked the code base, and your changes seem not to introduce a regression. If there is no `siteSlug`, the other three properties will not exist since we are taking siteSlug from the URL.

Thanks for making this changing and unifying SiteDetails type.

Reply:
Hi folks, I was testing the import content feature earlier today and realized it return as Page not found. If we choose to import the content only there's a case that value of `fromSite`  is empty. Please see the video below to replicate the issue.

https://user-images.githubusercontent.com/4074459/186574097-e316c965-c4eb-43fe-9845-03f104b60085.mov

Reply:
@Automattic/caribou, does [the issue noted above](https://github.com/Automattic/wp-calypso/pull/66729#discussion_r954487087) need to be addressed/fixed urgently? I don't see a relevant PR, but I can help get this moving again if needed, though I'll likely need some guidance to confirm how to test and verify the fix.

Reply:
The bug ticket is here: https://github.com/Automattic/wp-calypso/issues/66959
I'll fix it ASAP.

Reply:
💖 Thanks, @bogiii! Let me know if I can help with the issue!